### PR TITLE
credentials-email

### PR DESCRIPTION
**Pull request by saggineumann**

### CODE REVIEW

File: docs/getting-started/dashboard.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ sidebar_position: 10
 Use the **Credentials** tab to:
 
 1. Access your SFTP server information (host, usernames, and passwords). 
-2. Add or remove credentials.
-3. Manage access options for credentials (home directories, permissions, public keys, inbound network rules, and rotate passwords).
+2. Add, edit, seasrch or remove credentials.
```
Comment (motymichaely) [inline docs/getting-started/dashboard.md:9]:
```suggestion
2. Add, edit, search or remove credentials.
```

File: docs/getting-started/dashboard.md [hunk 2]
```diff
@@ -6,8 +6,8 @@ sidebar_position: 10
...
+3. Manage access options for credentials (home directories, permissions, public keys, inbound network rules, passwords and email association).
```
Comment (motymichaely) [inline docs/getting-started/dashboard.md:10]:
```suggestion
3. Manage access options for credentials (home directories, permissions, public keys, inbound network rules, passwords and email associations).
```

File: docs/getting-started/organization-settings.md [hunk 3]
```diff
@@ -11,6 +11,8 @@ To manage your organization, security, billing, and team settings, click **Setti
 
 SFTP To Go allows several authentication methods, which are ways to identify the client when they are logging into the server:
 
+* Magic Code - Identification based on username and unique, six character long, one-time-use codes, sent to the email address associated with the username. This form of authentication is only supported by the web portal.
```
Comment (motymichaely) [inline docs/getting-started/organization-settings.md:14]:
```suggestion
* Magic Code - identification based on username and a unique, six character one-time-use code sent to the email address associated with the username. This form of authentication is only supported by the web portal.
```

File: docs/how-to-connect/connect-web-portal.md [hunk 4]
```diff
@@ -5,16 +5,21 @@ sidebar_position: 10
...
 
-To access the web portal, first enable it in your organization's [settings tab](../getting-started/organization-settings#web-portal).
+To access the web portal, first ensure it is enabled in your organization's [settings tab](../getting-started/organization-settings#web-portal).
 Then, log in using your organization's portal login link, which can be found in the settings tab:
 
-1. Enter a credentials username and password.
-2. Click the "Log in" button.
+1. Enter a credentials username
+2. Enter a credentials password (or magic code if the credentials are associated with an email address and the [Magic Code](../getting-started/organization-settings#authentication-methods). authentication method is enabled)
+3. Click the "Log in" button.
```
Comment (motymichaely) [inline docs/how-to-connect/connect-web-portal.md:13]:
```suggestion
3. Click the "Sign in" button.
```

File: docs/how-to-connect/connect-web-portal.md [hunk 5]
```diff
@@ -5,16 +5,21 @@ sidebar_position: 10
...
 The web portal enables users with SFTP To Go credentials to manage files in their designated storage (i.e., their home directory) using a web browser over a secure HTTPS channel.
 
-To access the web portal, first enable it in your organization's [settings tab](../getting-started/organization-settings#web-portal).
+To access the web portal, first ensure it is enabled in your organization's [settings tab](../getting-started/organization-settings#web-portal).
 Then, log in using your organization's portal login link, which can be found in the settings tab:
 
-1. Enter a credentials username and password.
-2. Click the "Log in" button.
+1. Enter a credentials username
+2. Enter a credentials password (or magic code if the credentials are associated with an email address and the [Magic Code](../getting-started/organization-settings#authentication-methods). authentication method is enabled)
```
Comment (motymichaely) [inline docs/how-to-connect/connect-web-portal.md:12]:
```suggestion
2. Enter a credentials password or click "Email sign-in code" if the credentials are associated with an email address and the [Magic Code](../getting-started/organization-settings#authentication-methods) authentication method is enabled).
```

**Review by motymichaely — CHANGES_REQUESTED** [review]
@saggineumann I left a few comments and change suggestions.

**Comment by saggineumann** [comment]
Thanks @motymichaely - all of your changes look good

**Review by motymichaely — APPROVED** [review]

**Pull request merged**